pymbar/mbar_solvers.py

```diff
@@ -3,7 +3,12 @@
 import numpy as np
 import math
 import scipy.optimize
-from pymbar.utils import ensure_type, logsumexp, check_w_normalized
+from pymbar.utils import ensure_type, check_w_normalized
+import jax
+from jax.scipy.special import logsumexp
+from jax.ops import index_update, index
+from jax.config import config; config.update("jax_enable_x64", True)
```

Comment:
This will slow things down dramatically, esp. on the GPU. Do you really need 64bit precision?

Reply:
Probably yes - the algorithm doesn't seem to converge well with floats.  I can poke around and see what the deal is.

Reply:
Still need to work on this . . .